IoTBlockly/IoTBlocklyBackgroundApp/public/blockly/javascript-custom.js

```diff
@@ -12,7 +12,7 @@ Blockly.JavaScript['device_forever'] = function(block) {
 Blockly.JavaScript['device_pause'] = function(block) {
     // Pause statement.
     var pause = Blockly.JavaScript.valueToCode(block, 'PAUSE', Blockly.JavaScript.ORDER_ASSIGNMENT) || '100';
-    return 'basic.pause(' + pause + ');\n';
+    return 'pauseHelper(' + pause + ');\n';
```

Comment:
same as above. pauseHelper is injected into the code that comes across from index.html, so we're ok here